core/model/src/main/resources/bindings.xjb

```diff
@@ -217,9 +212,10 @@
                     if (this.isEmpty()) {
                         return new java.util.ArrayList<>();
                     }
-                    java.util.List<License> allLic = getLeftStatement().getLicenses();
-                    allLic.addAll(getRightStatement().getLicenses());
-                    return allLic;
+                    return java.util.stream.Stream.of(getLeftStatement().getLicenses(), getRightStatement().getLicenses())
+                            .flatMap(java.util.Collection::stream)
+                            .distinct()
+                            .collect(java.util.stream.Collectors.toList());
```

Comment:
why not collect to  a set, instead of `.distinct().collect(java.util.stream.Collectors.toList());`.

Reply:
Becausee we want to keep the return type of getLicenses